File: Python/Daily_Code_Challenge/260323-has-no-repeats.py
def has_no_repeats(s):

    for letter in range(1,len(s)):
        if s[letter] == s[letter-1]:
            return False

    return True



has_no_repeats("The quick brown fox jumped over the lazy dog.")


'''
No Consecutive Repeats
Given a string, determine if it has no repeating characters.

A string has no repeats if it does not have the same character two or more times in a row.
Tests:
Passed:1. has_no_repeats("hi world") should return True.
Passed:2. has_no_repeats("hello world") should return False.
Passed:3. has_no_repeats("abcdefghijklmnopqrstuvwxyz") should return True.
Passed:4. has_no_repeats("freeCodeCamp") should return False.
Passed:5. has_no_repeats("The quick brown fox jumped over the lazy dog.") should return True.
Passed:6. has_no_repeats("Mississippi") should return False.

'''

File: JavaScript/DailyCodeChallenge/260323_hasNoRepeats.js
function hasNoRepeats(str) {

  for (let x = 1; x<=str.length; x++) {
    if (str[x] == str[x-1]) {
      return false
    }
  }

  return true;
}


hasNoRepeats("The quick brown fox jumped over the lazy dog.")


/*
No Consecutive Repeats
Given a string, determine if it has no repeating characters.

A string has no repeats if it does not have the same character two or more times in a row.
Tests:
Passed:1. hasNoRepeats("hi world") should return true.
Passed:2. hasNoRepeats("hello world") should return false.
Passed:3. hasNoRepeats("abcdefghijklmnopqrstuvwxyz") should return true.
Passed:4. hasNoRepeats("freeCodeCamp") should return false.
Passed:5. hasNoRepeats("The quick brown fox jumped over the lazy dog.") should return true.
Passed:6. hasNoRepeats("Mississippi") should return false
*/